opa/opa.go

```diff
@@ -299,7 +323,17 @@ func NewInProcessAuthorizer(query string, paths []string, opts ...Option) (rbac.
 		o(c)
 	}
 
-	r := rego.New(rego.Query(query), rego.Load(paths, nil))
+	var r *rego.Rego
+
+	regoArgs := make([]func(*rego.Rego), 0, len(regoFunctions)+2)
```

Comment:
Why `+2`?

Reply:
@matej-g `# of custom functions` + 2 ( the query and loading the paths args ), the reason why we moved out from dynamic slice is to make the lint happy.

Reply:
Ah, you're right, now I see it below.